feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModel.kt

```diff
@@ -146,6 +149,12 @@ internal class SyncInfoViewModel @Inject constructor(
     }
 
     private fun load() = viewModelScope.launch {
+        // if device is not safe stop all sync operations
+        // users can root the device after login and we need to stop sync operations
+        if (isDeviceSafe().not()) {
```

Comment:
The background sync will still work no?

Reply:
@ybourgery  No, I chose the easiest way to disable the sync.
The load method calculates the needed info to be up or down-synced. if we break from this method nothing will be synced

Reply:
@ybourgery  I found I should also stop sync in EventSyncMasterWorker. will implement it now